spec/expeditor/rich_future_spec.rb

```diff
@@ -226,14 +226,12 @@
           max_threads: 1,
           max_queue: 1,
         )
-        future1 = Expeditor::RichFuture.new(executor: executor) do
-          42
-        end
-        future2 = Expeditor::RichFuture.new(executor: executor) do
-          42
+        futures = 3.times.map do
+          future1 = Expeditor::RichFuture.new(executor: executor) do
```

Comment:
(question) Why assign `future1` variable?

(nits) I prefer `Array.new(3)` instead of `3.times.map` to explain that building an array of 3 elements.